pdfarranger/exporter.py

```diff
@@ -50,12 +50,40 @@ def make_tmp_file(tmpdir):
     return f, filename
 
 
+def _normalize_rectangle(rect):
+    """
+    PDF Specification 1.7, 7.9.5, although rectangles are conventionally
+    specified by their lower-left and upper-right corners, it is acceptable to
+    specify any two diagonally opposite corners. Applications that process PDF
+    should be prepared to normalize such rectangles in situations where
+    specific corners are required.
+    """
+    rect = [float(x) for x in rect]
+    if rect[0] > rect[2]:
+        rect[0], rect[2] = rect[2], rect[0]
+    if rect[1] > rect[3]:
+        rect[1], rect[3] = rect[3], rect[1]
+    return rect
+
+
+def _intersect_rectangle(rect1, rect2):
+    return [
+        max(rect1[0], rect2[0]),
+        max(rect1[1], rect2[1]),
+        min(rect1[2], rect1[2]),
```

Comment:
Should this be?
`min(rect1[2], rect2[2]),`
`min(rect1[3], rect2[3]),`

Reply:
😱 yes